Description: added the generation of re2c files in the bootstrap
Author: Gary Kramlich <grim@reaperworld.com>
---
Index: ninja-1.3.3/bootstrap.py
===================================================================
--- ninja-1.3.3.orig/bootstrap.py	2013-05-24 21:38:28.000000000 -0500
+++ ninja-1.3.3/bootstrap.py	2013-05-24 21:38:28.000000000 -0500
@@ -64,6 +64,12 @@ except OSError:
     if e.errno != errno.EEXIST:
         raise
 
+# generate re2c files
+re2c_files = ['src/depfile_parser.in.cc', 'src/lexer.in.cc']
+for re2c in re2c_files:
+    out = re2c.replace('.in', '')
+    run(['re2c', '-b', '-i', '--no-generation-date', '-o', out, re2c])
+
 sources = []
 for src in glob.glob('src/*.cc'):
     if src.endswith('test.cc') or src.endswith('.in.cc'):
